src/components/common/styles/AutoComplete.scss

```diff
@@ -42,15 +46,16 @@ input[type='submit'] {
   right: 0;
 }
 
-.autocomplete-items div {
-  padding: 10px;
+.autocomplete p {
+  padding: 1%;
   cursor: pointer;
```

Comment:
Great UX addition. Shows immediate feedback to user that they can select city they hover over